File: app/src/main/java/com/fernandez/pablo/la24gnc/Model/EspecificacionProducto.java
package com.fernandez.pablo.la24gnc.Model;

/**
 * Created by pablo on 04/03/2017.
 */

public class EspecificacionProducto {

    private int codigo;
    private int rubro;
    private String descripcion;
    private double precio;

    public EspecificacionProducto(){

    }

    public EspecificacionProducto(int codigo, String descripcion, double precio, int rubro) {
        this.codigo = codigo;
        this.descripcion = descripcion;
        this.precio = precio;
        this.rubro = rubro;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    public int getRubro() {
        return rubro;
    }

    public void setRubro(int rubro) {
        this.rubro = rubro;
    }

    @Override
    public String toString() {
        return "EspecificacionProducto{" +
                "codigo=" + codigo +
                ", rubro=" + rubro +
                ", descripcion='" + descripcion + '\'' +
                ", precio=" + precio +
                '}';
    }
}

File: app/src/main/java/com/fernandez/pablo/la24gnc/Model/EspecificacionProducto.kt
package com.fernandez.pablo.la24gnc.Model

/**
 * Created by pablo on 04/03/2017.
 */

class EspecificacionProducto {

    var codigo: Int = 0
    var rubro: Int = 0
    var descripcion: String? = null
    var precio: Double = 0.toDouble()
    var urlImagen: String? = null

    constructor() {}

    constructor(codigo: Int, descripcion: String, precio: Double, rubro: Int) {
        this.codigo = codigo
        this.descripcion = descripcion
        this.precio = precio
        this.rubro = rubro
    }

    override fun toString(): String {
        return "EspecificacionProducto{" +
                "codigo=" + codigo +
                ", rubro=" + rubro +
                ", descripcion='" + descripcion + '\'' +
                ", precio=" + precio +
                '}'
    }
}
